### PR TITLE
Add 2 donut charts on the vaccination page for IC and hospital

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
This is already made with the new data keys in mind, so I've disabled the feature flags for now. Once the data is available it is ready to be QA'd therefore some screenshots with not the real data.

Desktop:
Screenshot 2021-11-01 at 16 30 31

Tablet:
Screenshot 2021-11-01 at 16 30 57

Mobile:
Screenshot 2021-11-01 at 16 30 44
Screenshot 2021-11-01 at 16 30 46

### CODE REVIEW

File: packages/app/src/pages/landelijk/vaccinaties.tsx [hunk 1]
```diff
@@ -219,58 +245,128 @@ const VaccinationPage = (props: StaticProps<typeof getStaticProps>) => {
...
+                    intensiveCareAmount: formatNumber(
+                      lastIntensiveCareValueVaccinationStatus.total_amount_of_people
+                    ),
+                    intensiveCareDateStart: IntensiveCareDateFromText,
+                    intensiveCareDateEnd: IntensiveCareDateToText,
+                  }
+                )}
+              >
+                <Box display="flex" flexDirection={{ _: 'column', sm: 'row' }}>
+                  <Box width="100%" display="flex" flexDirection="column">
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard16) [reply packages/app/src/pages/landelijk/vaccinaties.tsx:269]: I can understand if you use it with `space-between` to push the metadata to the bottom but you seem to use the spacer for that already.

**Review by VWSCoronaDashboard16 — COMMENTED** [review]
Looks good! Quite some nits and looking at the build it seems like you have to adjust the data properties & feature flag names on some other pages as well

File: packages/app/src/components/pie-chart.tsx [hunk 2]
```diff
@@ -143,45 +156,72 @@ export function PieChart<T>({
...
+                />
+              </Box>
+            ))}
+          </Box>
+          {
+            /**
+             * Check also for empty link text, so that clearing it in Lokalize
+             * actually removes the link altogether
+             */
+            link && !isEmpty(link.text) && (
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard14) [reply packages/app/src/components/pie-chart.tsx:211]: This doesn't work for typescript it's either checking the length of the string or using `isEmpty`.

File: packages/app/src/pages/landelijk/vaccinaties.tsx [hunk 3]
```diff
@@ -135,9 +146,24 @@ const VaccinationPage = (props: StaticProps<typeof getStaticProps>) => {
 
   const vaccineCoverageEstimatedLastValue =
     data.vaccine_coverage_per_age_group_estimated.last_value;
-  const lastValueVaccinationStatus =
+
+  const lastIntensiveCareValueVaccinationStatus =
     data.intensive_care_vaccination_status.last_value;
 
+  const lastHositalValueVaccinationStatus =
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard14) [reply packages/app/src/pages/landelijk/vaccinaties.tsx:153]: Oops, you are right. That was my original intend.

File: packages/app/src/components/pie-chart.tsx [hunk 4]
```diff
@@ -36,6 +46,9 @@ export function PieChart<T>({
   padAngle = 0.03,
   minimumPercentage = 0.5,
   icon,
+  isVerticalLayout,
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard17) [reply packages/app/src/components/pie-chart.tsx]: Don't entirely agree there. It's a convention in the codebase to prefix boolean props with `is` or `has`, making them easier to recognize. I think `hasVerticalLayout` would be best here

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Pull request merged**